libs/pipelines/api_v2_pipeline.py

```diff
@@ -120,6 +120,7 @@ def generate_metrics_and_latest(
     timeseries: OneRegionTimeseriesDataset,
     rt_data: Optional[OneRegionTimeseriesDataset],
     icu_data: Optional[OneRegionTimeseriesDataset],
+    log,
```

Comment:
That's funny because I saw the implicit passing around of a bound logger as an anti-pattern. What we have right now is not very consistent :-( which is why I've been erring towards keeping everything explicit on the stock. Cleaning up logging to be more consistent seems like a fine idea but for now I'd like to avoid introducing yet another way to do logging in this repo.

Reply:
Heh, that's why I'm still not sold on structlog :) never really had a problem with the std logger before haha